lib/apiservers/engine/backends/container.go

```diff
@@ -549,9 +559,7 @@ func (c *Container) containerStart(name string, hostConfig *container.HostConfig
 	if bind {
 		e := c.findPortBoundNetworkEndpoint(hostConfig, endpoints)
 		if err = c.mapPorts(portmap.Map, hostConfig, e); err != nil {
-			err = fmt.Errorf("error mapping ports: %s", err)
-			log.Error(err)
-			return InternalServerError(err.Error())
+			return InternalServerError(fmt.Sprintf("error mapping ports: %s", err))
```

Comment:
the caller is logging the error but I can add the logging back here if you want. It's just this is the only log call in the entire function :)